docs/Virtual-Environments.md

```diff
@@ -13,6 +13,9 @@ wrapper](https://gist.github.com/benlubas/5b5e38ae27d9bb8b5c756d8371e238e6). I w
 recommend a wrapper script of some kind if you are a python dev. If you're just installing these
 deps to use Molten with a non-python kernel, you can skip the wrapper without much worry.
 
+If you are using Anaconda or Miniconda as virtual environment manager, remind that your environment
+path is `CONDA_PREFIX` instead of `VIRTUAL_ENV`.
+
```

Comment:
This addition is unnecessary I think. It doesn't really need to be here

Reply:
OK. I also added it much consideration because I was confused whether molten support conda or not. However it looks more clear without it. I'll remove it!